actions/core/README.md

```diff
@@ -349,4 +363,4 @@ Please note that not every call can be visible in `invocation log` entry.
 | transition: `_error`   |  Yes (error level) |
 | TIMEOUT                |  No  |
 | Failure                |  No  |
-| Exception              |  No  |
\ No newline at end of file
+| Exception              |  No  |
```

Comment:
Is it valid?

Reply:
Please add some comment about handling errors here. HTTP action should not end with an exception when processing fails.